src/services/smartthings/lib/client.js

```diff
@@ -5,29 +5,38 @@ export default class SmartThingsClient {
     this._token = token;
   }
 
-  async _request(url, body) {
+  async _request(url, data) {
```

Comment:
Can probably revert all this, now we know the source of the 403 (old device in table)